kafka-ui-react-app/src/components/ConsumerGroups/Details/ResetOffsets/ResetOffsets.tsx

```diff
@@ -277,7 +284,10 @@ const ResetOffsets: React.FC = () => {
             buttonSize="M"
             buttonType="primary"
             type="submit"
-            disabled={selectedPartitions.length === 0}
+            disabled={
+              Object.values(errors).length > 0 ||
```

Comment:
We should use `isValid` instead of checking `Object.values(errors).length > 0`
https://react-hook-form.com/api/useform/formstate